docs/learn/learn-xcvm.md

```diff
@@ -39,12 +118,45 @@ enum Instruction {
 }
 ```
 
-A `MultiAsset` is a general identifier for an asset. It may represent both fungible and non-fungible
-assets, and in the case of a fungible asset, it represents some defined amount of the asset.
+- A `MultiAsset` is a general identifier for an asset. It may represent both fungible and
+  non-fungible assets, and in the case of a fungible asset, it represents some defined amount of the
+  asset.
+
+- A `MultiLocation` is a relative identifier, meaning that it can only be used to define the the
+  relative path between two locations, and cannot generally be used to refer to a location
+  universally.
+
+`TransferAsset` is one of the many instructions that can be executed within an XCM. For more
+information, please read [XCM Instructions in the wiki](./learn-xcm-instructions.md).
+
+## Locations in XCM
+
+XCM's abstract nature allows for locations to be relatively abstract notions that point to where but
+also _to who_ a particular action may affect. The `MulitLocation` type is what XCM uses to define
+these locations. A `MultiLocation` is a relative identifier that defines a **relative** path into
+some state-bearing consensus system.
+
+It is used to define the relative path between two locations, and cannot generally be used to refer
+to a location universally. It is very much akin to how a **relative** filesystem path works and is
+dependent on the context on which consensus system the message is being sent from, and what
+consensus system it is being sent to be executed.
+
+![XCM MultiLocation](../assets/cross-consensus/multilocation.png)
+
+`MultiLocation` has two primary fields:
+
+- A series of paths, called `Junctions`, which define an interior portion of state to descend into
+  it (sometimes called a "sub-consensus" system, such as a smart contract or pallet). An interior
+  location may also be used to refer to a Junction, used in the context of "a parachain is an
+  interior location of the relay chain", or how a UTXO is interior to Bitcoin's consensus.
+- The number of parent junctions at the beginning of a `MultiLocation`'s formation - in other words,
+  the number of parent consensus systems above it.
+
+There are a number of various `Junction` variants that may be used to describe a particular
+location - whether it's a 32 byte account, a Substrate pallet, or a pluralistic body.
+
```

Comment:
I would also add an example of a multilocation, e. g:

If you were on Parachain X on Polkadot and wanted to reference an account on Statemint (para chain id 1000) you would write the following `MultiLocation`:

```
../Parachain(1000)/AccountId32(<some_id>)
```
which in rust would be:

```rust
MultiLocation {
  parents: 1,
  interior: X2(Parachain(1000), <some_id>.into())
}
```

As illustrated by the `../` or `parents: 1` in Rust, this `MultiLocation` will go "up" by one consensus system and then go "down" the two junctions `Parachain` and `AccountId32`.
Keep in mind this location is relative, if this `MultiLocation` was interpreted by a Polkadot para chain, it would go "up" to the relay chain and then "down" to statemint (para chain id 1000) and to an account with some id.
If it was interpreted in Kusama, the underlying location it identifies would be different.
Some consensus systems might not even be able to interpret this location (Ethereum for example).

Reply:
This is great, I'll encapsulate this 🎉